blog/_posts/2024-08-07-eks-to-aks-series-part1.md

```diff
@@ -0,0 +1,33 @@
+
+We’re excited to announce the first in a new documentation series designed specifically for customers already using Amazon EKS, to help them replicate or migrate their workloads to AKS: [Replicate an AWS event-driven workflow (EDW) workload with KEDA and Karpenter in Azure Kubernetes Service (AKS)](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview)
+
+Moving a workload from another managed Kubernetes offering to AKS can be challenging. This series aims to provide a holistic, end-to-end experience by example to help ease that process. The samples in this series begin with infrastructure and code on EKS and end with equivalently functional infrastructure and code on AKS, while explaining the conceptual differences between AWS and Azure throughout.
+
+## EKS to AKS - Event Driven Workflow Scenario
+
+This new [article](https://learn.microsoft.com/en-us/azure/aks/eks-edw-overview) guides you through replicating an Amazon Web Services (AWS) Elastic Kubernetes Service (EKS) event-driven workflow (EDW) workload using [KEDA](https://keda.sh/) and [Karpenter](https://karpenter.sh/) in Azure Kubernetes Service (AKS). In the spirit keeping the workload as portable as possible, this guide prioritizes minimizing the changes needed to get the workload running on Azure.
+
+In this first article, the workload implements the competing consumers pattern with a producer/consumer app, facilitating efficient data processing by separating data production from data consumption. KEDA is used to scale pods running consumer processing, while AKS managed Karpenter autoscale Kubernetes nodes through the new AKS [Node Autoprovisioning (NAP)](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision) feature currently available in preview.
+
+You can either clone the AWS EKS sample and follow along with the guide, or read the guide to understand the necessary changes and the reasons behind them as outlined below.
+
+1. Understand Platform Differences: Review the differences between EKS and AKS in terms of services, architecture, and deployment for this workload.
+2. Re-architect the Workload: Analyze the existing EKS workload architecture and identify components that need changes to fit AKS.
+3. Update Application Code: Refactor elements of the code to make the app compatible with Azure APIs, services, and authentication models.
+4. Prepare for Deployment: Modify the EKS deployment process to use the Azure CLI.
+5. Deploy the Workload: Deploy the replicated EKS workload in AKS and test to ensure it functions as expected.
+
+## Summary
+
+Learn how to migrate Amazon EKS workloads to Azure AKS using KEDA and Karpenter in this comprehensive guide. Gain insights into the platform differences, re-architect the workload, update application code, prepare for deployment, and deploy the workload in AKS.
```

Comment:
This is kind of a dry ending and I don't think the blog post is long enough to really need a recap. Can we close with something more forward looking, like about what additional guides are coming? Maybe encourage people to reach out somehow with other ideas?